File: C/프로그래머스/0/120839. 가위 바위 보/가위 바위 보.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// 파라미터로 주어지는 문자열은 const로 주어집니다. 변경하려면 문자열을 복사해서 사용하세요.
char* solution(const char* rsp) {
 // return 값은 malloc 등 동적 할당을 사용해주세요. 할당 길이는 상황에 맞게 변경해주세요.
    
    int len = (int )strlen(rsp);
    char* answer = (char*)malloc(len);
    
    int idx = 0;
    for (int i = 0; i < len; i++) {
        switch (rsp[i]) {
            case '0':
                answer[idx++] = '5';
                break;
            case '2':
                answer[idx++] = '0';
                break;
            case '5':
                answer[idx++] = '2';
                break;
        }
    }
    answer[idx] = '\0';
    return answer;
}

File: 프로그래머스/lv0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String answer = "";
        String[] split = rsp.split("");
        
        for(int i = 0; i < split.length; i++) {
            switch (split[i]) {
                case "2": 
                    answer += "0";
                    break;
                case "0":
                    answer += "5";
                    break;
                case "5":
                    answer += "2";
                    break;
            }
        }
        
        return answer;
    }
}